src/app.ts

```diff
@@ -1,11 +1,4 @@
-import {TermLoader} from "./utils/term-loader";
+import {ProcessingManager} from "./ProcessingManager";
 
-TermLoader.loadFromDB((err, entities) => {
-    if (err) {
-        console.log("SHIT!\n", err);
-        return;
-    }
-    console.log("\n\n\n\n");
-    console.log("entities:\n", entities);
-
-});
\ No newline at end of file
+// fingers crossed, hope this works :3
+ProcessingManager.run();
```

Comment:
@nyxathid can you please test this as you're the only one with a working dummy DB.